src/odemis/gui/comp/canvas.py

```diff
@@ -171,7 +172,7 @@ def ignore_if_disabled(f, self, *args, **kwargs):
 class BufferedCanvas(wx.Panel):
     """ Abstract base class for buffered canvasses that display graphical data """
 
-    __metaclass__ = ABCMeta
+    __metaclass__ = no_conflict.classmaker()
```

Comment:
This probably drops the `ABCMeta` completely. From the `classmaker` documentation, it seems it should be `no_conflict.classmaker(right_metas=(ABCMeta,))`.